packages/documentation/src/stories/components/alert.stories.tsx

```diff
@@ -0,0 +1,392 @@
+import React from 'react';
+import { Meta, Story, Args } from "@storybook/react";
+import parse from 'html-react-parser';
+import { useArgs } from '@storybook/client-api';
+import docsPage from './alert.docs.mdx';
+
+import '@swisspost/design-system-styles/basics.scss';
+import '@swisspost/design-system-styles/components/alert.scss';
+import '@swisspost/design-system-styles/components/icons.scss';
+
+export default {
+  title: 'Components/Alert',
+  parameters: {
+    docs: {
+      page: docsPage
+    },
+    controls: {
+      exclude: [
+        'Title',
+        'Content'
+      ]
+    }
+  },
+  args: {
+    title: 'Titulum',
+    content: '<p>Contentus momentus vero siteos et accusam iretea et justo.</p>',
+    variant: 'alert-primary',
+    icon: 'null',
+    noIcon: false,
+    dismissible: false,
+    fixed: false,
+    action: false,
+    show: true
+  },
+  argTypes: {
+    title: {
+      name: 'Title',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    content: {
+      name: 'Content',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    variant: {
+      name: 'Variant',
+      description: 'Defines a style variant.',
+      control: {
+        type: 'select',
+        labels: {
+          'alert-primary': 'Primary',
+          'alert-success': 'Success',
+          'alert-danger': 'Danger',
+          'alert-warning': 'Warning',
+          'alert-info': 'Info'
+        }
+      },
+      options: [
+        'alert-primary',
+        'alert-success',
+        'alert-danger',
+        'alert-warning',
+        'alert-info'
+      ],
+      table: {
+        category: 'Variant',
+        type: {
+          summary: 'cssClass'
+        },
+        defaultValue: {
+          summary: 'alert-{variant}'
```

Comment:
As it stands, the "Default" column in the documentation table doesn't make much sense. It would be nice to add the "." in front of the summary to specify that these are CSS classes and even to change the title of the column if it is possible with storybook.

![image](https://user-images.githubusercontent.com/33580481/204849932-506da0ae-7240-4710-8f42-68344dab69f8.png)

Reply:
I have already updated the table.